Patches/turk's guns/Patch_Guns.xml

```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+	<success>Always</success>
+	<mods>
+		<li>turk's guns</li>
+	</mods>
+	
+	<match Class="PatchOperationSequence">
```

Comment:
CE now uses some fairly standard melee stats for various "classes" of ranged weapons (long-gun, pistol, heavy) just to make things easier and to allow some patch operations to be combined. Please check out how it's set up in Weapons_Guns.xml under Patches > Core and format this one similiarly.

Reply:
it is done